addons/mil_opcom/fnc_INS_helpers.sqf

```diff
@@ -270,11 +270,13 @@ ALiVE_fnc_INS_ied = {
                     _trg = createTrigger ["EmptyDetector",_pos];
                     _trg setTriggerArea [_size + 250, _size + 250,0,false];
                     _trg setTriggerActivation ["ANY","PRESENT",true];
+                    _num = ceil(_size/100);
                     _trg setTriggerStatements [
                         "this && {(vehicle _x in thisList) && ((getposATL _x) select 2 < 25)} count ([] call BIS_fnc_listPlayers) > 0",
-                            format["null = [getpos thisTrigger,%1,%2,%3] call ALIVE_fnc_createIED",_size,str(_id),ceil(_size/100)],
+                            format["null = [getpos thisTrigger,%1,%2,%3] call ALIVE_fnc_createIED",_size,str(_id),_num],
                             format["null = [getpos thisTrigger,%1] call ALIVE_fnc_removeIED",str(_id)]
                     ];
+                    ["storeTrigger", [_size,_id,_pos,_num]] call ALiVE_fnc_IED;
```

Comment:
Need to add a check above to confirm if persistence has been set on `ALiVE_MIL_IED`